neo/public/javascripts/common/services.js

```diff
@@ -1,12 +1,157 @@
 angular.module('cloudberry.common', [])
   .service('Asterix', function($http, $timeout, $location) {
```

Comment:
Changing the name will lead to a fail in front-end. There is no map appeared.
`$http` can be eliminated. But since I didn't find where the function is called, I am not sure if the removal will influence the performance. Based on my simple test on local machine, it is good.